doc/base/classes.xml

```diff
@@ -53158,40 +53158,44 @@
 			<return type="void">
 			</return>
 			<description>
-				Start the timer.
+				Start the Timer.
 			</description>
 		</method>
 		<method name="stop">
 			<return type="void">
 			</return>
 			<description>
-				Stop (cancel) the timer.
+				Stop (cancel) the Timer.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="autostart" type="bool" setter="set_autostart" getter="has_autostart" brief="">
+			If [code]true[/code], Timer will automatically start when entering the scene tree. Default value: [code]false[/code].
 		</member>
 		<member name="one_shot" type="bool" setter="set_one_shot" getter="is_one_shot" brief="">
+			If [code]true[/code], Timer will stop when reaching 0. If [code]false[/code], it will restart. Default value: [code]false[/code].
 		</member>
 		<member name="process_mode" type="int" setter="set_timer_process_mode" getter="get_timer_process_mode" brief="" enum="Timer.TimerProcessMode">
+			Processing mode. Uses TIMER_PROCESS_* constants as value.
```

Comment:
Related to #11137. We also need references to enums. In this case it would be `[enum Timer.TimerProcessMode]` or `[enum TimerProcessMode]` (we could make it search the enum in the class before searching for global enums when the class name is not specified).